docs/eventing/broker/alternate/README.md

```diff
@@ -6,3 +6,7 @@ implementations provided by Knative Eventing.
 
 ## GCP Broker
 The GCP Broker is optimized for running in GCP. For more details, refer to the [doc](https://github.com/google/knative-gcp/blob/master/docs/install/install-gcp-broker.md).
+
+## Apache Kafka Broker
+
+The Kafka Broker is an alternative Kafka Broker. For more details, refer to the [doc](../kafka-broker.md).
```

Comment:
Alternative to what? :slightly_smiling_face: 
I actually don't think these descriptions add much, same for GCP (like, we'd assume it's optimized for GCP from the name? :woman_shrugging: ) so I would dare to say we could probably just have "For information about the Apache Kafka broker, see [link]"

Reply:
Done: https://github.com/knative/docs/pull/2740/commits/34fc316e29963fdc9dc70cb8d0173d33cd1297a3